packages/@apphosting/adapter-nextjs/src/utils.ts

```diff
@@ -30,42 +36,46 @@ export const isMain = (meta: ImportMeta) => {
   return process.argv[1] === fileURLToPath(meta.url);
 };
 
-export function populateOutputBundleOptions(cwd: string): OutputBundleOptions {
-  const outputBundleDir = join(cwd, ".apphosting");
+export function populateOutputBundleOptions(rootDir: string, appDir: string): OutputBundleOptions {
+  const outputBundleDir = join(rootDir, ".apphosting");
+  // In monorepo setups, the standalone directory structure will mirror the structure of the monorepo.
```

Comment:
Is this only the case for monorepo setups?

What about the following project setup where `appdir = /nested/app/src`?
```
/
| nested
- | app
  - | src
```

Perhaps we should make this comment a bit more general to any non-"/" appdir setups

Reply:
Yes it should support the project setup you described, although I should note that there are some rough edges to smooth out to support these non-monorepo nested setups.